.github/workflows/reusable-web-e2e.yml

```diff
@@ -24,6 +24,10 @@ jobs:
       # leaving the Dashboard hanging ...
       # https://github.com/cypress-io/github-action/issues/48
       fail-fast: false
+      matrix:
+        # run 5 copies of the current job in parallel
+        containers: [1, 2, 3, 4, 5]
+        total: [5]
```

Comment:
Bringing back the Cypress parallel tests execution and also sharding for the Playwright. 
Before we had this option turned on for Cypress, but we decided to disable it with the hope that the tests would be less flaky, but it didn't help. So I decided to turn it back.